broker/core/src/config/applier/endpoint.cc

```diff
@@ -231,7 +231,9 @@ void endpoint::_discard() {
 
   // Stop multiplexing.
   try {
-    multiplexing::engine::instance().stop();
+    auto eng = multiplexing::engine::instance_ptr();
+    if (eng)
+      eng->stop();
```

Comment:
again, why is it mandatory to check eng?

Reply:
removed

Reply:
removed